### DISCUSSION TITLE
[V5] CORS error on default API setup

### DISCUSSION THREAD

**Discussion by briodev** [adonisjs/core#1935] [Help]
Fresh install of v5. Chose "API Server"  Using default cors.ts file with origin: true Also tried origin: '*'

When posting from react app (localhost)  I get the cors error:
 "Access to XMLHttpRequest at 'localhost:3333/test' from origin 'http://localhost:3000' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, chrome-untrusted, https."

Is there a step I'm missing to allow access?

**Comment by thetutlage** [comment]
What's the request method?

**Reply by thetutlage** [reply]
In fact, it will be nice to share all the request headers

**Reply by briodev** [reply]
Thanks for getting back. Method is POST (using Axios from React app on localhost:3000)
I am not setting headers in Axios
If I look in Chrome Dev Tools -> Network -> Headers

```
Provisional headers are shown
Accept: application/json, text/plain, */*
Content-Type: application/json;charset=utf-8
Referer: http://localhost:3000/
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36
```

Do I need to set headers in the React app? New to this but I'm just trying to test a POST request and console log the response from clean install of Adonis v5 and Create React App

Pretty sure I have to be missing something.  Maybe because it's all running on localhost?

Thanks again!

**Reply by RomainLanz** [reply]
What's the content of `config/cors.ts` ?

**Comment by tiagocdr** [comment]
Was there any solution to this ? Having the exact same problem
